src/sweeper/workspace_info.py

```diff
@@ -26,3 +44,49 @@ def get_featureclasses(workspace_path):
                 fc_list.append(fc)
 
         return fc_list
+
+
+#: A function to return a list of feature classes based on the change detection table.
+def get_change_detection():
+    '''
+    workspace_path: full path to the change detection table.
```

Comment:
**chore**: this parameter comment needs to be updated or removed.

Reply:
Done